packages/mongo/remote_collection_driver.ts

```diff
@@ -121,10 +121,25 @@ MongoInternals.defaultRemoteCollectionDriver = once((): RemoteCollectionDriver =
 
   // Initialize database connection on startup
   Meteor.startup(async (): Promise<void> => {
-    await driver.mongo.client.connect();
+    return new Promise((resolve, reject) => {
+      driver.mongo.client
+        .connect()
+        .then(resolve)
+        .catch((error) => {
+          // Fix an issue with development mode crashing on idle
+          // https://github.com/meteor/meteor/issues/13108
+          if (
+            Meteor.isDevelopment &&
+            error?.message?.includes("PoolClearedOnNetworkError")
+          ) {
+            return;
+          }
+          reject(error);
+        });
+    });
```

Comment:
I will adapt it to use `try/catch` with `await`. Initially, it wasn't working as intended, as it was ignoring all MongoDB errors instead of just `PoolClearedOnNetworkError`.